lib/iris/util.py

```diff
@@ -23,7 +22,6 @@
 import numpy as np
 import numpy.ma as ma
 
-from iris._deprecation import warn_deprecated
```

Comment:
I think a bit of a merge conflict has happened here. It looks like warn_deprecated is now being used since https://github.com/SciTools/iris/pull/4514 was merged in, so this shouldn't be removed.

I think the original tests would have passed since 4514 hadn't been merged in yet? So it didn't catch this

Reply:
Ah, I thought that at the time but missed it as a consequence of the merge. Fixed now